privacyidea/lib/tokenclass.py

```diff
@@ -1620,12 +1617,13 @@ def generate_symmetric_key(self, server_component, client_component,
         server component with bytes of the client component.
                 
         :param server_component: The component usually generated by privacyIDEA
-        :type server_component: hex string
+        :type server_component: str
```

Comment:
I would like to preserve the information, that the expected string is in hex format. Then we should at least put this in the docstring.

Reply:
fixed in 933d699